src/System.Drawing.Graphics/src/System/Drawing/Graphics/Image.cs

```diff
@@ -0,0 +1,75 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+
+
+namespace System.Drawing.Graphics
+{
+    public enum PixelFormat
+    {
+        ARGB,
```

Comment:
These should follow our casing conventions. See https://msdn.microsoft.com/en-us/library/ms229043(v=vs.110).aspx